apps/bynder/src/index.js

```diff
@@ -244,7 +244,7 @@ setup({
       type: 'List',
       value: 'MultiSelect,SingleSelectFile',
       default: 'MultiSelect',
-      description: '"SingleSelectFile" allows you to select a specific derivative.',
+      description: '"MultiSelect is the best choice for most customers. If you specifically need access to transformations, use SingleSelectFile. Note that you will need to change your frontend to use transform base URLs to load images in SingleSelectFile mode."',
```

Comment:
Nice. I have just a few minor comments:

> If you specifically need access to transformations

Maybe make it clear that we mean the "dynamic" ones that you create on the fly.

I had a few other minor wording things. I'll wrap it all in a suggestion.